contracts/marketplace/src/execute.rs

```diff
@@ -964,10 +964,11 @@ pub fn execute_remove_stale_ask(
     // A CW721 approval will be removed when
     // 1 - There is a transfer or burn
     // 2 - The approval expired (CW721 approvals can have different expiration times)
-    let res = Cw721Contract::<Empty, Empty>(collection.clone(), PhantomData, PhantomData).owner_of(
+    let res = Cw721Contract::<Empty, Empty>(collection.clone(), PhantomData, PhantomData).approval(
         &deps.querier,
         token_id.to_string(),
-        false,
+        env.contract.address.to_string(),
+        None,
```

Comment:
maybe just set include expired to `false` so that it's explicit

Reply:
thanks modified other places where we weren't using explicit calls